txbits/conf/evolutions/default/2.sql

```diff
@@ -895,7 +892,8 @@ begin
   select confirmation_token = a_token into strict success from withdrawals where id = a_id;;
 
   if success then
-    update withdrawals set user_rejected = true where id = a_id;;
+    update withdrawals set user_rejected = true where id = a_id returning * into strict w;;
+    perform withdrawal_delete(w.amount, w.user_id, w.currency, w.fee);;
```

Comment:
This function should be renamed since `withdrawal_delete` doesn't actually delete the withdrawal, it refunds it.